shared_model/cryptography/blob.hpp

```diff
@@ -20,30 +20,65 @@
 
 #include "interfaces/model_primitive.hpp"
 #include "utils/string_builder.hpp"
+#include "utils/lazy_initializer.hpp"
 
 namespace shared_model {
   namespace crypto {
 
+    static const std::string code = {'0',
+                                     '1',
+                                     '2',
+                                     '3',
+                                     '4',
+                                     '5',
+                                     '6',
+                                     '7',
+                                     '8',
+                                     '9',
+                                     'a',
+                                     'b',
+                                     'c',
+                                     'd',
+                                     'e',
+                                     'f'};
+
     /**
      * Blob interface present user-friendly blob for working with low-level
      * binary stuff. Its length is not fixed in compile time.
      */
     class Blob : public interface::ModelPrimitive<Blob> {
+     private:
+      template <typename Value>
+      using Lazy = detail::LazyInitializer<Value>;
+
      public:
+      Blob(std::string data) : data_(data), hex_([this] {
+        std::string res(size() * 2, 0);
```

Comment:
use `std::stringstream` & `std::hex` instead

Reply:
like that?
```
std::stringstream ss;
ss << data_ << std::
return ss.str();
```